src/launcher.rs

```diff
@@ -93,5 +136,6 @@ pub fn ruby_command() -> Command {
   env::args().skip(1).foreach(|a| {
     command.arg(a);
   });
+  command.args(args.as_slice());
```

Comment:
This addition is a bit confusing in the context of the undocumented env::args pulling behavior of this method. I'm thinking we have two sensible options:

1. Require consumer to provide env args on their own
2. Document the behavior of this method more thoroughly

I'm leaning toward (2), so heres what I might add to the documents
```
/// Build a command to execute the Ruby Cucumber Server
/// Takes a list of extra command line arguments to the server
/// NOTE: This command also passes the command line arguments passed to this executable.
```

Leaving the exact choice and wording to you.